ember-cli-build.js

```diff
@@ -15,6 +15,13 @@ module.exports = function(defaults) {
     }
   });
 
+  app.import("node_modules/moment/locale/de.js");
+  app.import("node_modules/moment/locale/de-ch.js");
+  app.import("node_modules/moment/locale/it.js");
+  app.import("node_modules/moment/locale/it-ch.js");
+  app.import("node_modules/moment/locale/fr.js");
+  app.import("node_modules/moment/locale/fr-ch.js");
```

Comment:
Does this affect consuming apps or just the dummy app?

Reply:
If I understood Ember CLI correctly, this is for all apps. But I'd have to test it to be sure.